avm/res/insights/metric-alert/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -35,9 +35,7 @@ module nestedDependencies 'dependencies.bicep' = {
   scope: resourceGroup
   name: '${uniqueString(deployment().name, location)}-nestedDependencies'
   params: {
-    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
     actionGroupName: 'dep-${namePrefix}-ag-${serviceShort}'
-    location: location
```

Comment:
Same comment as before. For now we keep `location` in and may remove it on scale if the linter warning is taken care of.

Reply:
In this case, the only dependency is an action group, which is a 'global' resource and doesn't use the location parameter. Keeping it generates a linter warning `Parameter "location" is declared but never used..`. Should we keep it anyway?

Reply:
No, that's just my bad. I thought somehow the location was removed from the test deployment